lib/std/fs.zig

```diff
@@ -818,11 +832,6 @@ pub const Dir = struct {
         /// `true` means the opened directory can be scanned for the files and sub-directories
         /// of the result. It means the `iterate` function can be called.
         iterate: bool = false,
-
-        /// `true` means the opened directory can be passed to a child process.
-        /// `false` means the directory handle is considered to be closed when a child
-        /// process is spawned. This corresponds to the inverse of `O_CLOEXEC` on POSIX.
-        share_with_child_process: bool = false,
```

Comment:
Why are we getting rid of this? Having something that maps to `CLOEXEC` seems sensible to me.

Reply:
Because of the race condition that you helpfully pointed out: https://github.com/ziglang/zig/issues/5190#issuecomment-622668195

Reply:
There are still situations where you want to `open` without CLOEXEC; e.g. opening a file as stdin/stdout (which is done to implement shell redirection).